scripts/build-monorepo.sh

```diff
@@ -462,6 +467,12 @@ process_workspace_webpack_config () {
     done
 }
 
+use_node_version_from_nvmrc () {
+    source ~/.nvm/nvm.sh
+    nvm install < .nvmrc
+    nvm use < .nvmrc
```

Comment:
I don't think `nvm` uses `stdin` at all, so I believe these redirects are unnecessary. As long as `.nvmrc` is present in the current working directory, this should work the same:
```suggestion
    nvm install
    nvm use
```